docker/pygeoapi-config.yml

```diff
@@ -26,70 +26,31 @@ logging:
 metadata:
     identification:
         title:
-            en: WIS 2.0 in a box
+            en: wis2box-api
         description:
             en: |-
-                WIS 2.0 in a box provides a platform with the capabilities
+                WIS 2.0 in a box (wis2box) provides a platform with the capabilities
                 for centres to integrate their data holdings and publish them
                 to the WMO Information System with a plug and play capability
-                supporting data publishing, discovery and access
+                supporting data publishing, discovery and access.
+                The wis2box-api provides processes to manage datasets and tools
```

Comment:
WIS 2.0 in a box (wis2box) provides a platform with the capabilities
for centres to integrate their data holdings and publish them
to the WMO Information System with a plug and play capability
supporting data publishing, discovery and access.
wis2box-api provides an API providing data and processes for interactive visualization, data transformation and publication.